_studio/mfx_lib/decode/vp8/include/mfx_vp8_dec_decode_hw.h

```diff
@@ -218,7 +218,7 @@ class VideoDECODEVP8_HW : public VideoDECODE, public MfxCriticalErrorHandler
     {
         VideoDECODEVP8_HW* decoder;
         mfxFrameSurface1* surface_work;
-        FrameMemID memId;
+        UMC::FrameMemID memId;
```

Comment:
Well, that's a separate change actually.

Reply:
ok, I will extract it to separate commit